hack/components/bump-kubemacpool.sh

```diff
@@ -45,6 +45,9 @@ metadata:
 spec:
   template:
     spec:
+      affinity: AFFINITY
```

Comment:
This patch is called "cnao_image_patch". Would you please introduce a new patch for placement?

Reply:
Ah, I see! Fixed

Reply:
ops, nope. mistake was made

Reply:
now it's fixed